nextjs_tdd/src/app/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Metadata } from 'next';
+import './globals.css';
+import ContextWrapper from '@/context';
+
+export const metadata: Metadata = {
+    title: 'Airline Booking'
+};
+
+export default function RootLayout({
+    children,
+    home,
+    airlines
+}: Readonly<{
+    children: React.ReactNode;
+    home: React.ReactNode;
+    airlines: React.ReactNode;
+}>) {
+    return (
+        <html lang="en">
+            <body>
+                <ContextWrapper>
```

Comment:
If you are planning to manage the state globally this is fine, but if you are looking at specific pages, probably moving to nearest parent component would be good.

Reply:
Resolved

Reply:
Please mark the comment as resolved once the comment is addressed.

Reply:
Sure